tests/acceptance/features/coreApiWebdavPreviews/previews.feature

```diff
@@ -140,11 +140,11 @@ Feature: previews of files downloaded through the webdav API
 
   Scenario: preview content should be updated if the file content is updated (content with UTF chars)
     Given user "Alice" has uploaded file "filesForUpload/lorem.txt" to "/lorem.txt"
-    And user "Alice" has uploaded file with content "सिमसिमे पानी" to "/lorem.txt"
+    And user "Alice" has uploaded file with content "ओनक्लाउड फाएल शेरिङ्ग एन्ड सिन्किङ" to "/lorem.txt"
```

Comment:
increased the content so that there is something visible in thumnail (preview image). See `unicode-fixture.png` file